USFMToolsSharp/Models/Markers/QMarker.cs

```diff
@@ -27,6 +28,25 @@ public override string PreProcess(string input)
             typeof(TLEndMarker),
             typeof(WMarker),
             typeof(WEndMarker),
+            typeof(VMarker),
         };
+
+        public override bool TryInsert(Marker input)
+        {
+            if (input is VMarker && Contents.Any(m => m is VMarker))
+            {
+                return false;
+            }
+            if(Contents.Count > 0 && Contents[Contents.Count - 1].TryInsert(input))

```

Comment:
This block (lines 40-49) looks like it's repeated from the parent method.  

Consider calling `return base.TryInsert(input);` instead of repeating the code.

Reply:
Yeah since it is pretty much the same that would be the best way of doing that.

Reply:
I didn't think of that